openslides/poll/models.py

```diff
@@ -66,10 +66,13 @@ def print_weight(self, raw=False):
             percent_base = 0
         return print_value(self.weight, percent_base)
 
-
 PERCENT_BASE_CHOICES = (
-    {'value': 'WITHOUT_INVALID', 'display_name': ugettext_lazy('Only all valid votes')},
-    {'value': 'WITH_INVALID', 'display_name': ugettext_lazy('All votes cast (including invalid votes)')},
+    {'value': 'WITHOUT_ABSTAIN', 'display_name':  ugettext_lazy(
+        'Only Yes and No votes')},
+    {'value': 'WITHOUT_INVALID', 'display_name': ugettext_lazy(
+        'Only all valid votes (Yes/No/Abstain)')},
```

Comment:
As I suggested in my last comment: Please remove "Only" from this string: "All valid votes (Yes/No/Abstain)" 
Should be consistent with the "All votes cast (..)" option